charts/housnav/values-uat.yaml

```diff
@@ -4,7 +4,7 @@ image:
   name: house-innovations-housnav-web
   repository: ""
   tag: ""
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
Why did we change this to 'always' ?

Reply:
This resolves the issue with HOUSNAV-107. The Helm chart does not deploy a new image version with the same Docker tag when pullPolicy is set to "IfNotPresent". When we set pullPolicy to "Always", the Helm chart will upgrade the image even if the Docker tag name remains unchanged. This is a necessary condition for our Docker tag strategy.